components/layout/Header.js

```diff
@@ -159,20 +164,24 @@ function Header() {
                   Contact
                 </Link>
 
-                <Link
-                  href="/"
-                  className="tw-text-xl tw-p-2 tw-font-inter tw-text-gray-500  hover:tw-text-gray-500 tw-font-medium"
-                >
-                  Sign In
-                </Link>
+                {!(isUserLoggedIn || isMentorLoggedIn) && (
+                  <Link
+                    href="/"
+                    className="tw-text-xl tw-p-2 tw-font-inter tw-text-gray-500  hover:tw-text-gray-500 tw-font-medium"
+                  >
+                    Sign In
+                  </Link>
+                )}
+
+                {/* theme icon */}
                 <SunIcon className="tw-h-10 tw-bg-white tw-rounded-2xl tw-px-0.5 tw-w-9 tw-text-yellow-400 tw-cursor-pointer" />
-                <button
-                  type="button"
-                  onClick=""
-                  className=" tw-w-full tw-rounded-lg tw-border btn:hover btn:active tw-bg-[#845ec2] hover:tw-scale-[0.99] active:tw-scale-[1.01] active:tw-shadow-none tw-py-3 tw-text-white tw-transition-all tw-text-center tw-text-md tw-font-inter tw-flex tw-items-center tw-justify-center"
-                >
-                  Signup
-                </button>
+
+                {/* show profile card if user is logged in else show signup button */}
+                {isUserLoggedIn || isMentorLoggedIn ? (
+                  <UserProfile />
+                ) : (
+                  <ButtonLink text="Sign Up" href="/" />
+                )}
```

Comment:
For mobile: can we have profile icon , not in drop down... instead near the hamburger. with the same behavior when click on profile that card appears.
<img width="418" alt="Screenshot 2023-07-20 at 12 35 06 PM" src="https://github.com/anmode/grabtern-frontend/assets/88332977/e6cd7e42-b9f9-404d-ab9d-b671569f8de4">

Reply:
Do we need signup button near hamburger?

Reply:
> Do we also need signup button near hamburger?

No no...it's ok in hamburger menu